README.md

```diff
@@ -1,137 +1,106 @@
+ - Send raw gcode commands to the printer (and see the response).
 
- - Send print jobs to the printer
+ - Low-level access to MQTT, PPPP and HTTPS APIs.
 
- - Stream camera image/video to your computer
+ - Send print jobs (gcode files) to the printer.
 
+ - Stream camera image/video to your computer.
 
-## Upcoming and planned features
+ - Easily monitor print status.
 
- - Easily monitor print status
+### Upcoming and Planned Features
 
- - Integration into other software. Home Assistant? Prusa Slicer?
+ - Integration into other software. Home Assistant? Cura plugin?
 
-Pull requests always welcome :-)
+Let us know what you want to see; Pull requests always welcome! :smile:
 
-## Installation instructions
+## Installation
 
-First, please follow the [installation
-instructions](documentation/example-file-usage/example-file-prerequistes.md) for
-your platform.
+There are currently two ways to do an install of ankerctl. You can install directly from git utilizing python on your Operating System or you can install from Docker which will install ankerctl in a containerized environment. Only one installation method should be chosen. 
 
-Verify that you can start `ankerctl.py`, and get the help screen:
+> **Warning**
+> Docker Installation ONLY works on Linux at this time
 
-### For Windows, use: 
-```powershell
-python3 ankerctl.py -h
-```
-If that does not work then try running using `python ankerctl.py -h`
-
-### Linux and MacOS, use:
-```sh
-./ankerctl.py -h
-```
-
-You should see the below output - if not then head back to [installation instructions](documentation/example-file-usage/example-file-prerequistes.md)
-```
-Usage: ankerctl.py [OPTIONS] COMMAND [ARGS]...
-
-Options:
-  -k, --insecure  Disable TLS certificate validation
-  -v, --verbose   Increase verbosity
-  -q, --quiet     Decrease verbosity
-  -h, --help      Show this message and exit.
-
-Commands:
-  config  View and update configuration
-  http    Low-level http api access
-  mqtt    Low-level mqtt api access
-  pppp    Low-level pppp api access
-```
-
-Before you can use ankerctl, you need to import the configuration.
+Follow the instructions for a [docker install](./documentation/install-from-docker.md) (recommended) or a [git install](./documentation/install-from-git.md).
 
-Support for logging in with username and password is not yet supported, but an
-`auth_token` can be imported from the saved credentials found in `login.json` in
-Ankermake Slicer. See `ankerctl.py config import -h` for details:
+## Usage
 
-```
-Usage: ankerctl.py config import [OPTIONS] path/to/login.json
+### CLI Utilities (Python Scripts)
 
-  Import printer and account information from login.json
+1. Import your AnkerMake account data by opening a terminal window in the folder you placed ankerctl in and running the following command:
 
-  When run without filename, attempt to auto-detect login.json in default
-  install location
+   ```sh
+   ./ankerctl.py config import [OPTIONS] path/to/login.json 
```

Comment:
This is a bit of a mix between an actual command to run, and a metasyntactical template for one.

Maybe like so:

```sh
   ./ankerctl.py config import path/to/login.json
```

("config import" doesn't currently take any options anyway..)